modules/network-observability-resource-recommendations.adoc

```diff
@@ -0,0 +1,20 @@
+//module included in the following assemblies:
+// network_observability/configuring_operator.adoc
+
+:_content-type: REFERENCE
+[id="network-observability-resource-recommendations_{context}"]
+= Resource management and performance recommendations
+
+The amount of resources required by Network Observability depends on the size of your cluster and your requirements for ingesting and storing observability data. There are settings to consider that can help you get an optimal setup and observability.
```

Comment:
Sorry this phrase might be interpreted that a user ingests and stores observability data :). How about:

> The amount of resources required by Network Observability depends on the size of your cluster and the cluster's requirements for ingesting and storing observability data.

Reply:
Thanks @stleerh and @dfitzmau for helping clarify this bit. Addressed to Steven's recommended wording.